inst/stories.yaml

```diff
@@ -1,4 +1,14 @@
 # Please add stories at the top ------------------------------------------
+SLV-S005: 
```

Comment:
In the merge conflict at 7513cf7c (Merge branch 'develop' into refactor/param, 2022-08-16), this should have been bumped to SLV-S006.  And this will have yet another conflict with PR #1008.  Unfortunately, in-tree stories very much breed conflicts :/